### PR TITLE
Feature/items bulk import

### PR DESCRIPTION
**Pull request by jsconan**
Add a CLI tool that imports items packages from directory.
Useful to import easily bunch of items without wasting time on the UI, like for CAT items.

### CODE REVIEW

File: scripts/cli/importItems.php [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+ *
+ * @package oat\taoQtiItem\scripts\cli
+ * @author Jean-Sébastien Conan <jean-sebastien@taotesting.com>
+ */
+class importItems implements Action, ServiceLocatorAwareInterface
+{
+    use OntologyAwareTrait;
+    use ServiceLocatorAwareTrait;
+    
+    const ITEM_CLASS_URI = 'http://www.tao.lu/Ontologies/TAOItem.rdf#Item';
```
Comment (antoinerobin) [inline scripts/cli/importItems.php]: we already have the taoOntology::ITEM_CLASS_URI

File: scripts/cli/importItems.php [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+    protected $processed = 0;
+
+    /**
+     * @param array $params
+     * @return Report
+     * @throws \common_Exception
+     */
+    public function __invoke($params = [])
+    {
+        \common_ext_ExtensionsManager::singleton()->getExtensionById('taoQtiItem');
```
Comment (antoinerobin) [inline scripts/cli/importItems.php]: do you really need it ?

Comment (jsconan) [reply scripts/cli/importItems.php]: It does not seem indeed. I removed it

File: scripts/cli/importItems.php (hunk 1 above)
Comment (jsconan) [reply scripts/cli/importItems.php]: I was aware of that. Fixed now

**Comment by jsconan** [comment]
Changes made.

**Pull request merged**